### PR TITLE
Update README.md: improved grammar

### PR DESCRIPTION
**Pull request by xternet**

### CODE REVIEW

**Review by xinbenlv — DISMISSED** [review]
LGTM. Thank you for contribution. @xternet

**Review by Pandapip1 — APPROVED** [review]

**Comment by Pandapip1** [comment]
Waiting on editor approval.

**Review by SamWilsn — APPROVED** [review]

**Comment by SamWilsn** [comment]
@eip-bot rerun

**Pull request merged**
